charts/opentelemetry-collector/UPGRADING.md

```diff
@@ -16,6 +16,34 @@ created but the Memory Ballast Extension will still be removed.
 Depending on the progress made in [Issue 891](https://github.com/open-telemetry/opentelemetry-helm-charts/issues/891),
 the use of `GOMEMLIMIT` may completely replace the Memory Ballast Extension in the future.
 
+## 0.87.2 to 0.88.0
+
+As part of working towards using the [OpenTelemetry Collector Kubernetes Distro](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-k8s) by default, the chart now requires users to explicitly set an image repository and command name. If you are already explicitly setting an image repository and command name this breaking change does not affect you.
+
+If you are using a OpenTelemetry Community distribution of the Collector we recommend you use `otel/opentelemetry-collector-k8s`, but carefully review the [components included in this distribution](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-k8s/manifest.yaml) to make sure it includes all the components you use in your configuration. In the future this distribution will become the default image used for the chart.
+
+You can use the OpenTelemetry Collector Kubernetes Distro by adding these lines to your values.yaml:
+
+```yaml
+image:
+  repository: "otel/opentelemetry-collector-k8s"
+
+command:
```

Comment:
is command required to be set? I would opt for not setting it if we can avoid it.

Reply:
[from the docs](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#define-a-command-and-arguments-when-you-create-a-pod): command corresponds to entrypoint, and if someone is using a collector distro, each one of them should have an accurate entrypoint

Reply:
Since we set `command` in the container spec instead of `args` we have to set a command.  

If we want to take advantage of `ENTRYPOINT` in the containers, which I think is a good idea, we'll need to switch to using `args`, which impacts how `command.extraArgs` and `command.name` works for users who are overriding it with their own image or args.

Reply:
feels like a good follow up to me, I don't think it's required here

Reply:
I'll try to make a separate PR that handles that transition as I think I'll improve this PR's transition.

Reply:
Created https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1149

Reply:
Updated to make command name optional again